python/ray/util/client/server/server.py

```diff
@@ -556,7 +584,11 @@ def default_connect_handler(job_config: JobConfig = None):
         logs_servicer=logs_servicer,
         grpc_server=server,
     )
-    server.start()
+    loop = asyncio.get_event_loop()
+    loop.run_until_complete(server.start())
+    t = threading.Thread(
```

Comment:
might need to make it daemon=True so it won't block the client_server from shutdown

Reply:
Done!